web/src/components/logs-table.css

```
@@ -132,3 +132,46 @@
+.lv-plugin__virtualized-table--export-csv-button {
+  margin-bottom: 20px;
+}
+
+.lv-plugin__virtualized-table .pf-v6-c-table .pf-v6-c-table__action {
+  padding-left: 0px !important;
+  width: 5% !important;
+
+  @media screen and (max-width: 992px) {
+    & {
+      width: 10% !important;
+    }
```

Stylelint (17.10.0)

[error] Redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)